src/test/java/org/numenta/nupic/network/LayerTest.java

```diff
@@ -307,7 +311,7 @@ public void testHalt() {
         Parameters p = NetworkTestHarness.getParameters().copy();
         p = p.union(NetworkTestHarness.getHotGymTestEncoderParams());
         p.set(KEY.RANDOM, new MersenneTwister(42));
-        p.set(KEY.AUTO_CLASSIFY, Boolean.TRUE);
+        p.set(KEY.AUTO_CLASSIFY, false);
```

Comment:
@Hopding 
Does this need to be explicitly set to false? What happens if these are not stated at all?

Also, shouldn't the test remain the same (i.e. test restarting with installed classifiers?)